pkg/controllers/monitoring-stack/controller.go

```diff
@@ -46,11 +46,13 @@ import (
 
 	"github.com/go-logr/logr"
 	monv1 "github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring/v1"
+	grafana_operator "github.com/rhobs/monitoring-stack-operator/pkg/controllers/grafana-operator"
```

Comment:
it would be nice if package names didn't use `_`

Reply:
Done